packages/api/src/command/medical/organization/create-organization.ts

```diff
@@ -62,6 +98,8 @@ async function createOrganizationInternal(
       organizationNumber,
       cxId,
       data: { name, type, location },
+      cqActive: false,
+      cwActive: orgType === "healthcare_provider" ? true : false,
```

Comment:
what does `cwActive` mean in this case now? why do we want only `healthcare_provider` to be active by default?

Reply:
Commonwell has an active flag just like Carequality -- so it just determines whether the CW organization created should be active or not. See line 51 and 70 in the same file.

As we discussed, for providers Commonwell should start active (currently how it is today) and only start inactive specifically for OBO facilities from IT vendors. Carequality should always start inactive.

These fields will be forever false for IT vendor organizations.

These are fields are on the organization table in RDS and I modeled them from off of the same-named ones from the facility table in RDS (which allowNull but not explicitly and are default false). Caveat is I made these ones allowNull explicitly false per the style guidelines.

Reply:
Update: both providers will start false and are only synced when approval is true.